docs/src/models/biological.md

```diff
@@ -1,166 +1,307 @@
 # Chemical Reaction Models
 
-The biological models functionality is provided by DiffEqBiological.jl and helps
-the user build discrete stochastic and differential equation based systems
+The biological models functionality is provided by `DiffEqBiological.j`l and helps
+the user to build discrete stochastic and differential equation based systems
 biological models. These tools allow one to define the models at a high level
 by specifying reactions and rate constants, and the creation of the actual problems
-is then handled by the modeling package.
+is then handled by the modelling package.
 
-## The Reaction DSL
+## The Reaction DSL - Basic 
+This section covers some of the basic syntax for building chemical reaction network models.
+#### Basic syntax
 
-The `@reaction_network` DSL allows you to define reaction networks in a more
+The `@reaction_network` macro allows you to define reaction networks in a more
 scientific format. Its input is a set of chemical reactions and from them it
 generates a reaction network object which can be used as input to `ODEProblem`,
 `SDEProblem` and `JumpProblem` constructors.
 
 The basic syntax is:
 
 ```julia
-rn = @reaction_network rType begin
+rn = @reaction_network begin
   2.0, X + Y --> XY               
-  1.0, XY --> Z            
+  1.0, XY --> Z1 + Z2            
 end
 ```
 
-where each line corresponds to a chemical reaction. The input `rType` designates
-the type of this instance (all instances will inherit from the abstract type
-`AbstractReactionNetwork`).
+where each line corresponds to a chemical reaction. Each reaction consists of a reaction rate (the expression on the left hand side of  `,`), a set of substrates (the expression in-between `,` and `-->`), and a set of products (the expression on the right hand side of `-->`). The substrates and the products may contain one or more reactants, separated by `+`.  The naming convention for these are the same as for normal variables in Julia.
 
-The DSL can handle several types of arrows, in both backwards and forward
-direction. If a bi-directional arrow is used two reaction rates must be
-designated. These two reaction networks are identical:
+The chemical reaction model is generated by the `@reaction_network` macro and stored in the `rn` variable (a normal variable, do not need to be called `rn`). The macro generates a differential equation model according to the law of mass action, in the above example the ODEs become:
+d[X]/dt = -2.0\*[X\]*[Y]
```

Comment:
Put this in literal form via ```, or use TeX

Reply:
You are right, that becomes much better. Can one use TeX in these documents?

Reply:
yes, you can use TeX. Just make it a ```math environment

Reply:
Ok, I think I got it to work:
$$
\frac{d[X]}{dt} = -2.0[X]\cdot [Y]\\
\frac{d[Y]}{dt} = -2.0[X]\cdot [Y]\\
\frac{d[XY]}{dt} = 2.0[X]\cdot [Y] - 1.0[XY]\\
\frac{d[Z1]}{dt}= 1.0[XY]\\
\frac{d[Z2]}{dt} = 1.0[XY]\\
$$
(very useful knowledge by the way :) )

Is there a way to update these pull requests without deleting and creating a new. Also, are your review automatically made as changes or do I have to make them happen to the PR?

Reply:
it emails me of all changes so I can look at it. If you commit and push to the branch you opened the PR from it will update the PR.

Reply:
Ok. I clicked on the "Edit on GitHub" link on the docs page (http://docs.juliadiffeq.org/latest/models/biological.html) and then the edit button and simply pasted the text in. I guess it is possible from there as well (although maybe I will try to do it the other way next time instead).

Either way, sending you the text.

Reply:
Your equation syntax is still off. Your example should be something like:
```math
\begin{align*}
\frac{\mathrm{d}\left[X\right]}{dt} =&  - 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] \\
\frac{\mathrm{d}\left[Y\right]}{dt} =&  - 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] \\
\frac{\mathrm{d}\left[XY\right]}{dt} =& 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] - \left[ XY \right] \\
\frac{\mathrm{d}\left[Z1\right]}{dt} =& XY \\
\frac{\mathrm{d}\left[Z2\right]}{dt} =& XY \\
\end{align*}
```
which is surrounded by ```math